tests/testthat/test-oauth.R

```diff
@@ -111,13 +111,19 @@ test_that("oob must be a flag", {
 })
 
 test_that("can not use oob in non-interactive session", {
+  old <- options()
+  on.exit(options(old))
+  options(rlang_interactive = FALSE)
+
```

Comment:
This test would previously fail when run with `test_active_file()`, so I fixed that while I was here.